projects/jupyter-server-ydoc/jupyter_server_ydoc/handlers.py

```diff
@@ -290,8 +290,15 @@ async def on_message(self, message):
         """
         On message receive.
         """
-        message_type = message[0]
+        if message == "save_to_disc":
+            try:
+                self.room._saving_document = asyncio.create_task(
+                    self.room._maybe_save_document(self.room._saving_document)
+                )
+            except Exception:
+                self.log.error("Couldn't save content from room: %s", self._room_id)
 
+        message_type = message[0]
```

Comment:
Picked up a conflict here as the code checking for chat message type was removed